script/tool/test/util.dart

```diff
@@ -23,6 +23,13 @@ import 'mocks.dart';
 
 export 'package:flutter_plugin_tools/src/common/repository_package.dart';
 
+/// The relative path from a package to the file that is used to enable
+/// README excerpting for a package.
+// This is a shared constant to ensure that both readme-check and
+// update-excerpt are looking for the same file, so that readme-check can't
+// get out of sync with what actually drives excerpting.
+const String kReadmeExcerptConfigPath = 'example/build.excerpt.yaml';
```

Comment:
nit: I don't think we typically start constants with a `k` in Dart. I'm assuming this the pattern for this package?

Reply:
I learned recently that the Flutter style guide disagrees with Dart style on this for global constants:

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k